soweego/importer/models/orm/bibsys_entity.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import String
+from sqlalchemy import Integer
+
+from .base_entity import BaseEntity
+
+class BibsysEntity(BaseEntity):
+    __tablename__ = "bibsys"
+
+    id = Column(Integer, ForeignKey('base_entity.id'), primary_key=True, autoincrement=True)
```

Comment:
Does this mean that
- `BaseEntity` will materialize a table in the DB;
- `BibsysEntity` will use a column of that table as a foreign key?
If so, this is the opposite of what we want, as it will already create a useless join.